jupyter-notebooks/analytics/user-guide/2. Analytic Feeds Results.ipynb

```diff
@@ -0,0 +1,1546 @@
+   "source": [
+    "We'll use the `links` property to get the link to the Subscription's **Feed** and then request the Feed from the API:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Construct the Feed request url from the Subscription link\n",
+    "seg_subscription_feed_url = list(filter(lambda link: link['rel'] == 'feed', seg_subscription['links']))[0]['href']\n",
+    "print(seg_subscription_feed_url)\n",
+    "\n",
+    "# Make the request for the Subscription's Feed\n",
+    "seg_subscription_feed = requests.get(seg_subscription_feed_url, auth=BASIC_AUTH).json()\n",
+    "seg_subscription_feed"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have our **Feed** data for the Subscription, let's use the `target` property to determine our `mosaic series`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the mosaics series id\n",
+    "mosaic_target_series_id = seg_subscription_feed['target']['config']['series_id']\n",
+    "mosaic_target_series_id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using the Mosaic Series Id, we can make a request to the Planet Mosaics API using the following endpoint `https://api.planet.com/basemaps/v1/series/{series_id}`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Construct the mosaics series url\n",
+    "mosaic_target_series_url = \"https://api.planet.com/basemaps/v1/series/\" + mosaic_target_series_id\n",
+    "print(mosaic_target_series_url)\n",
+    "\n",
+    "# Make a request to the Mosaics API\n",
+    "target_mosaic_series = requests.get(mosaic_target_series_url, auth=BASIC_AUTH).json()\n",
+    "target_mosaic_series"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see some metadata here about our series, including the `first_acquired` and `last_acquired` dates, the `interval` at which a new mosaic gets created. \n",
+    "\n",
+    "Now that we have the Mosaic series, we can see that there's a link to `mosaics` under the `_links` property. These are the individual raster segmentation mask \"mosaics\" made up of several smaller image files called \"quads\". The mosaics themselves are also available as `xyz` tile basemaps, served by Planet's tile servers. Let's make a final request to the `mosaics` link which will give us the urls we need to make a visualization of the Subscription's resuts!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make a request to get the subscription's mosaics\n",
+    "subscription_mosaics = requests.get(target_mosaic_series['_links']['mosaics'], auth=BASIC_AUTH).json()\n",
```

Comment:
The subscription mosaics will be the subset of these within the subscription time range of interest. To make the map below avoid showing months for which there are not results, you can filter the mosaics list here.
e.g.
```python
target_mosaics = requests.get(target_mosaic_series['_links']['mosaics'], auth=BASIC_AUTH).json()['mosaics']
subscription_mosaics = [m for m in target_mosaics if m['first_acquired'] >= subscription['startTime'] and m['last_acquired'] <= subscription['endTime']]
```

Reply:
Ah, I see you have a filtering example in a later section. It still might be nice to constrain what gets visualized in the first raster data demo map.